phy/utils/settings.py

```diff
@@ -94,8 +94,11 @@ class UserSettings(BaseSettings):
     def declare_namespace(self, namespace, scope='global'):
         self._store[scope][namespace] = Bunch()
 
-    def read_settings_file(self, path, file_namespace=None, scope='global'):
-        """Return a dictionary {namespace: {key: value}} dictionary."""
+    def load(self, path, file_namespace=None, scope='global'):
+        """ load a setting file """
```

Comment:
when github plan to generate commits for that kind of fix? Would be so much easier...

collaborative commits!